sciolyid/cogs/race.py

```diff
@@ -209,37 +183,28 @@ def __init__(self, bot):
                 await ctx.send("**Sorry, the maximum amount to win is 1 million.**")
                 limit = 1000000
 
-            logger.info(f"adding bw: {bw}; addon: {addon}; state: {state}; media: {media}; limit: {limit}")
+            logger.info(f"adding bw: {bw}; group: {group}; ")
 
             database.hmset(
                 f"race.data:{ctx.channel.id}", {
                     "start": round(time.time()),
                     "stop": 0,
                     "limit": limit,
                     "bw": bw,
-                    "state": state,
-                    "addon": addon,
-                    "media": media
+                    "group": group,
                 }
             )
-
             database.zadd(f"race.scores:{ctx.channel.id}", {str(ctx.author.id): 0})
-            await ctx.send(f"**Race started with options:**\n{await self._get_options(ctx)}")
-
-            if str(database.hget(f"race.data:{ctx.channel.id}", "media"))[2:-1] == "image":
-                logger.info("auto sending next bird image")
-                addon, bw = map(str, database.hmget(f"race.data:{ctx.channel.id}", ["addon", "bw"]))
-                birds = self.bot.get_cog("Birds")
-                await birds.send_bird_(ctx, addon[2:-1], bw[2:-1])
+            await ctx.send(f"**Race started with options:**\n{self._get_options(ctx)}")
 
-            if str(database.hget(f"race.data:{ctx.channel.id}", "media"))[2:-1] == "song":
-                logger.info("auto sending next bird song")
-                birds = self.bot.get_cog("Birds")
-                await birds.send_song_(ctx)
+            logger.info("auto sending next image")
+            media = self.bot.get_cog("Media")
+            await media.send_pic_(ctx, group, bw)
 
     @race.command(
         brief="- Views race",
-        help="- Views race.\nRaces allow you to compete with your friends to ID a certain bird first."
+        help="- Views race.\n" +
+        f"Races allow you to compete with your friends to ID a certain {config.options['id_type']} first."
     )
     @commands.cooldown(1, 3.0, type=commands.BucketType.channel)
     async def view(self, ctx):
```

Comment:
```suggestion
                await ctx.send("**There is no race in session.** *You can start one with `b!race start`*")
```
(on L219) should update prefixes to use the config